apps/api-documenter/src/documenters/MarkdownDocumenter.ts

```diff
@@ -4,11 +4,7 @@
 /* eslint max-lines: "off" */
 
 import * as path from 'path';
-import {
-  PackageName,
-  FileSystem,
-  NewlineKind
-} from '@rushstack/node-core-library';
+import { PackageName, FileSystem, NewlineKind } from '@rushstack/node-core-library';
```

Comment:
I do prefer `imports` with multiple imported entities being always wrapped. Too bad there isn't an option for this.

Reply:
FYI there are two GitHub issues related to this topic, and both of them are asking for the opposite:

https://github.com/prettier/prettier/issues/5253
https://github.com/prettier/prettier/issues/5995

The rationale seems to be "nobody wants to read a huge block of imports, just make it as small as possible".